dhis-2/dhis-api/src/main/java/org/hisp/dhis/user/PasswordValidationRule.java

```diff
@@ -34,6 +34,9 @@
  */
 public interface PasswordValidationRule
 {
+    String MANDATORY_PARAMETER_MISSING = "Mandatory parameter missing";
+    String I18_MANDATORY_PARAMETER_MISSING = "mandatory_parameter_missing";
```

Comment:
Add `static final` (eventually `protected static final`)

Reply:
aren't they static final by default?

Reply:
No. They are not. By default, they are only the package-private right now if you do not specify private/protected/public. But not static nor final.